docs/ARCHITECTURE.md

```diff
@@ -35,8 +35,8 @@ Backend manager creates a backend for the execution of the CNNNetwork. We implem
 * Basic backend
 * VAD-M backend
 
-Basic backend is used for Intel<sup>®</sup> CPUs, Intel<sup>®</sup> integrated GPUs and Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs). The backend creates an inference request and runs inference on a given input data.
+Basic backend is used for Intel<sup>®</sup> CPUs, Intel<sup>®</sup> integrated GPUs and Intel<sup>®</sup> Movidius™ Vision Processing Units (VPUs). The backend creates an inference request and runs inference on given input data.
 
-VAD-M backend is used for Intel® Vision Accelerator Design with 8 Intel<sup>®</sup> Movidius™ MyriadX VPUs (referred as VAD-M or HDDL). We support batched inference execution in the VAD-M backend. When the user provides a batched input, multiple inference requests are created and inference is run in parallel on all the available VPUs in the VAD-M.
+VAD-M backend is used for Intel® Vision Accelerator Design with 8 Intel<sup>®</sup> Movidius™ MyriadX VPUs (referred to as VAD-M or HDDL). We support batched inference execution in the VAD-M backend. When the user provides a batched input, multiple inference requests are created, and inference is run in parallel on all the available VPUs in the VAD-M.
```

Comment:
We use "batched inference execution" to refer that a separate inference will be executed asynchronously for each sample in the batch. But it may not be clear that we are executing separate parallel inferences for each sample. Should we use some other term like "batch level parallelism" or something else?

Reply:
Changed it as per the suggestion.